server/src/main/java/io/seata/server/console/controller/ConfigurationController.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.seata.server.console.controller;
+
+import io.seata.config.ConfigValidator;
+import io.seata.console.constant.Code;
+import io.seata.console.result.SingleResult;
+import io.seata.server.console.service.ConfigurationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Junduo Dong
+ */
+@RestController
+@RequestMapping("/api/v1/admin/configuration")
+public class ConfigurationController {
```

Comment:
The version of the API should also be reflected in the file structure.

Reply:
Done.